data/mods/ssb/moves.ts

```diff
@@ -196,6 +196,37 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		target: "normal",
 		type: "Ghost",
 	},
+
+	// phiwings99
+	ghostof1v1past: {
+		accuracy: true,
+		basePower: 0,
+		category: "Status",
+		desc: "Imprisons and traps the target, and then transforms into them.",
+		shortDesc: "Imprison + Mean Look + Transform.",
+		name: "Ghost of 1v1 Past",
+		pp: 1,
+		priority: 0,
+		flags: {},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Imprison', target);
+			this.add('-anim', source, 'Mean Look', target);
+			this.add('-anim', source, 'Transform', target);
+		},
+		volatileStatus: 'imprison',
```

Comment:
You should be applying thhe imprison to yourself or else it doesnt work

Reply:
O